content/en/docs/reference/labels-annotations-taints/audit-annotations.md

```diff
@@ -72,6 +72,30 @@ violated from the PodSecurity enforcement.
 See [Pod Security Standards](/docs/concepts/security/pod-security-standards/)
 for more information.
 
+## apiserver.latency.k8s.io/etcd
+
+Example: `apiserver.latency.k8s.io/etcd: "4.730661757s"`
+
+This annotation indiactes the measure of latency incurred inside the storage layer,
+it accounts for the time it takes to send data to the etcd and get the complete response back.
+
+Please note that it does not include the time incurred in admission, or validation.
+
+## apiserver.latency.k8s.io/decode-response-object
+
+Example: `apiserver.latency.k8s.io/decode-response-object: "450.6649ns"`
+
+This annotation records the decoding time of response object.
```

Comment:
_This annotation records the time taken to_ … (what goes here)?

Reply:
This annotation records the time taken to decode the response object.

Reply:
Is it one of these?
```suggestion
This annotation records the time taken to decode the response received from the storage layer (etcd).
```

```suggestion
This annotation records the time that elapses whilst the API server is decoding the response that
it is about to send.
```

?

Reply:
It's the first one . 
For reference : https://github.com/tnorlin/kubernetes/blob/851cf43a35862aeeafc4a0966f7d1e0836f675e4/staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go#L156

Reply:
```go
This is latency incurred inside the function that takes an object returned from the underlying storage layer (etcd) and performs decoding of the response object.
```